src/database/Connector.java

```diff
@@ -124,16 +124,44 @@ public boolean verifyAccount(String username, String password) {
             String passwordInDB = resultPassword.getString(1);
             // password does not match
             if (!passwordInDB.equals(password)) {
-              System.err.println("the password does not match");
+              //System.err.println("the password does not match");
               return false;
             }
-            System.out.println("the account exists");
+            //System.out.println("the account exists");
             return true;
           }
         }
       }
     } catch (SQLException e) {e.printStackTrace();return false;}
-    System.err.println("the account does not exist");
+    //System.err.println("the account does not exist");
+    return false;
+  }
+
+  /**
+   * verify if the account username input by users matches an account in the
+   * database
+   *
+   * @param username
```

Comment:
The description is missing from param username.
`@param username name of the user account to verify.`